docs build requires a github token (#3035)
chef/automate: components/automate-chef-io/content/docs/backup.md

We recommend that you configure a backup directory on a disk with sufficient free space.

> The default backup directory is `/var/opt/chef-automate/backups`. If it does not exist it will be created during deployment.

Reviewer: ```suggestion
The default backup directory is `/var/opt/chef-automate/backups`. Chef Automate creates this directory during deployment if it does not already exist.
```

> ## Storing Backups in a Single-file Archive for Airgapped Installations

Reviewer: ```suggestion
## Storing Backups in a Single-File Archive for Airgapped Installations
```

> The backup directory contains both the timestamp-based directory for the configuration, and the reporting data stored in the `automate-elasticsearch-data` directory.

Reviewer: ```suggestion
The backup directory contains both the timestamp-based directory for the configuration and the reporting data stored in the `automate-elasticsearch-data` directory.
```
I'm not sure that I understand the single-file aspect of this. Is the header correct and relevant?

You will need to archive both of these directory types in any offline backups you create.

> By default, the backup directory will be located at `/var/opt/chef-automate/backups`.

Reviewer: ```suggestion
By default, the backup directory is located at `/var/opt/chef-automate/backups`.
```
